CHANGELOG.md

```diff
@@ -78,8 +79,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Features
 
-* [#4829] (https://github.com/osmosis-labs/osmosis/pull/4829) Add highest liquidity pool query in x/protorev
-* [#4878] (https://github.com/osmosis-labs/osmosis/pull/4878) Emit backrun event upon successful protorev backrun
+* [#4829] (<https://github.com/osmosis-labs/osmosis/pull/4829>) Add highest liquidity pool query in x/protorev
```

Comment:
nit: can we avoid changing the old backlog links?

Reply:
very yes, and my apologies -- wonder what is up with my editor?